docs/overview/explore-with-script-lab.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Explore Office JavaScript API using Script Lab
+description: Learn about Script Lab and how you can use it to explore the Office JS API and prototype functionality.
+ms.topic: article
+ms.date: 04/03/2019
+localization_priority: Normal
+---
+
+# Explore Office JavaScript API using Script Lab
+
+As you begin or continue to develop your Office add-in, there are several ways to explore the Office JavaScript API (for example, quick starts and tutorials) but the [Script Lab add-in](https://aka.ms/getscriptlab), which is available free from the Office store, enables you to do so in a sort of sand-boxed environment. Script Lab is also a great way to prototype and verify functionality you want to include in your add-in.
+
+## What is Script Lab?
+
+Script Lab is a tool for anyone who wants to learn about developing Office Add-ins for Excel, Word, PowerPoint, and Project using Office JS API. It provides IntelliSense so you can see what's available. You can use existing samples in an Office host application to quickly try out something or use a sample as the base for your own. Another cool thing is that, through Script Lab, you might be able to try beta or preview functionality like [custom functions](/office/dev/add-ins/excel/custom-functions-overview).
```

Comment:
Consider going a step further than saying it has Intellisense and point out it uses the Monaco framework VS Code uses (or just that it's like VS Code).